internal/backend/projects/projects.go

```diff
@@ -114,3 +118,153 @@ func (ps *Projects) SetResources(ctx context.Context, projectID sdktypes.Project
 func (ps *Projects) DownloadResources(ctx context.Context, projectID sdktypes.ProjectID) (map[string][]byte, error) {
 	return ps.DB.GetProjectResources(ctx, projectID)
 }
+
+func (ps *Projects) Export(ctx context.Context, projectID sdktypes.ProjectID) ([]byte, error) {
+	const manifestFileName = "autokitteh.yaml"
+	hasManifest := false
```

Comment:
i think we should always export it, because stuff might have changed. the files will contain the original manifest, but we should also return the updated somehow as well.

Reply:
It's confusing since there are two `autokitteh.yaml` files in the zip.

Reply:
Renamed the original and added the current.